From b2dcabe6a4a155aa11d604f738341b2f11c6ed75 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Sat, 17 May 2014 11:06:15 +0200
Subject: [PATCH] Make TLBB the default skin

Signed-off-by: j1nx <p.steenbergen@j1nx.nl>
---
 xbmc/system.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/system.h b/xbmc/system.h
index 1f4e15f..af54d5f 100644
--- a/xbmc/system.h
+++ b/xbmc/system.h
@@ -274,5 +274,5 @@
 #if defined(HAS_SKIN_TOUCHED) && defined(TARGET_DARWIN_IOS) && !defined(TARGET_DARWIN_IOS_ATV2)
 #define DEFAULT_SKIN          "skin.touched"
 #else
-#define DEFAULT_SKIN          "skin.confluence"
+#define DEFAULT_SKIN          "skin.tlbb"
 #endif
-- 
1.9.1

